patches/minecraft/net/minecraft/client/gui/components/events/ContainerEventHandler.java.patch

```diff
@@ -0,0 +1,25 @@
+--- a/net/minecraft/client/gui/components/events/ContainerEventHandler.java
++++ b/net/minecraft/client/gui/components/events/ContainerEventHandler.java
+@@ -24,6 +_,8 @@
+    }
+ 
+    default boolean m_6375_(double p_94695_, double p_94696_, int p_94697_) {
++      boolean result = false;
```

Comment:
Should have a comment explaining that it fixes a vanilla bug `// Forge: Fix MC-147605`

Reply:
Added a comment right after a flag declaration